Client.xcodeproj/project.pbxproj

```diff
@@ -12071,6 +12162,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = E60961861B62B8A700DD640F /* Fennec.xcconfig */;
 			buildSettings = {
+				VALIDATE_WORKSPACE = YES;
```

Comment:
Can you explain this? I'm curious about why this works and builds when Carthage is still there for the others.